config/install/oe_authentication.settings.yml

```diff
@@ -0,0 +1 @@
+base_url: http://authentication:8001
```

Comment:
Let's also change the readme to say `authentication` instead of `oeauth-server` for the `etc/hosts` file.

Also, please add to the README that sites can override this base url in their settings.php file.